index.js

```diff
@@ -110,8 +110,8 @@ async function handleAlert(location, alertType) {
   }
 }
 
-async function sendSingleAlert(locationid, message) {
-  const location = await db.getLocationData(locationid)
+async function sendSingleAlert(locationid, message, client) {
+  const location = await db.getLocationData(locationid, client)
```

Comment:
Good find. That would have been a nasty one with things hanging for no apparent reason. I'm glad it was this `sendSingleAlert` that needed to change. I was worried it was the one in `brave-alert-lib` which would be more of a pain to change.